.travis/install-llvm-and-runtime-compiler.sh

```diff
@@ -4,7 +4,10 @@ set -ev
 if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
   sudo apt-get install -y llvm-${LLVM_VERSION} llvm-${LLVM_VERSION}-dev
 
-  sudo apt-get install -y llvm-${LLVM_VERSION}-tools clang-${LLVM_VERSION}
+  sudo apt-get install -y clang-${LLVM_VERSION}
+  if [ "${LLVM_VERSION}" != "3.8" ]; then
+      sudo apt-get install -y llvm-${LLVM_VERSION}-tools
```

Comment:
Why is llvm-3.8-tools not needed anymore?

Reply:
Hmm, this was needed before we started using llvm-toolchain repos. It should not be needed now. I will try to drop it.

Reply:
For 3.8, this is still needed, because:
```
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:
The following packages have unmet dependencies:
 llvm-3.8-tools : Depends: llvm-3.8-dev (= 1:3.8~svn278388-1~exp1) but 1:3.8-2ubuntu3~trusty5 is to be installed
E: Unable to correct problems, you have held broken packages.
```

Reply:
I see, thanks for having a look. Do you know if it's needed at all, considering llvm-3.8 works without it?

Reply:
@kren1 llvm-tools contains `FileCheck` and `not`, so you we need for testing.

Reply:
So this means no testing for llvm-3.8? I guess for those two programs we could use an different llvm-tools version?

Reply:
But tests are run and succeed AFAICT. So the tools are maybe in different package in 3.8?

Reply:
Actually, `FileCheck` and `not` are downloaded if not available, right?

Reply:
@jirislaby Yes, exactly.  And we can either use lit provided with LLVM or the python package.

Reply:
So what't the solution here? I feel like we don't need to special case this for 3.8.